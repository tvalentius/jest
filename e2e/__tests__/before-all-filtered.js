/**
 * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 *
 * @flow
 */
'use strict';

const runJest = require('../runJest');

describe('Correct BeforeAll run', () => {
  it('ensures the BeforeAll of ignored suite is not run', () => {
    const result = runJest('before-all-filtered');
    expect(result.stdout.replace(/\\/g, '/')).toMatchSnapshot();
  });
});
