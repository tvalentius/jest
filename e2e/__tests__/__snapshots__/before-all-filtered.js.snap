// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correct BeforeAll run ensures the BeforeAll of ignored suite is not run 1`] = `
"  console.log __tests__/before-all-filtered.test.js:3
    beforeAll 1

  console.log __tests__/before-all-filtered.test.js:6
    beforeEach 1

  console.log __tests__/before-all-filtered.test.js:15
    It Foo

  console.log __tests__/before-all-filtered.test.js:9
    afterEach 1

  console.log __tests__/before-all-filtered.test.js:12
    afterAll 1
"
`;
